applications/ALSAFullDuplex.C

```diff
@@ -32,6 +34,8 @@ int printUsage(string name, string devOut, string devIn, int latency) {
     cout<<"\t e.g. "<<name<<" [options] /tmp/out.wav"<<endl;
     cout<<"\t -D : The name of the output device : (-D "<<devOut<<")"<<endl;
     cout<<"\t -C : The name of the input  device : (-C "<<devIn<<")"<<endl;
+    cout<<"\t -R : The sample rate to operate at : (default 48000)"<<endl;
+    cout<<"\t -f : The sample format to use : (default S32_LE)"<<endl;
```

Comment:
Again, don't hard code, pass this in as a variable.
Also print the available formats : https://github.com/flatmax/gtkiostream/blob/master/include/ALSA/Hardware.H#L201